index.html

```diff
@@ -16,42 +16,246 @@
             <img src="images/humbger.png" alt="toggle menu" />
           </div>
         </div>
-      </nav></header>
-      <section class="headline container">
-        <div class="heading">
-          <h1>I’m Glory <br>Glad to see<br> you!</h1>
+      </nav>
+    </header>
+    <section class="headline container">
+      <div class="heading">
+        <h1>
+          I’m Glory <br />Glad to see<br />
+          you!
+        </h1>
+      </div>
+      <div class="paragraph-one">
+        <p class="paragraph">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="lets-connect"><a href="#">LET’S CONNECT</a></p>
+      </div>
+      <ul class="social-media-links">
+        <li>
+          <a href="#"><img src="images/twitter.png" alt="twitter" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Linkedinicon.png" alt="linkedin" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector.png" alt="microverse" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/gitgub.png" alt="github" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector3.png" alt="vector" /></a>
+        </li>
+      </ul>
+    </section>
+    <main>
+      <section class="grid-container works-section container">
+        <div>
```

Comment:
In the Figma design, there is some margin between the sections. Kindly put some margin between the two sections, they are currently stacked on top each other now 😎
<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/205440987-9b4a4b3c-8518-4098-b4b2-224d2934336f.png">

Reply:
- [ ] Great job on implementing the feedback given by the previous reviewer :100: